File: Python/Classificação de Triângulos.py
num1 = float (input())
num2 = float (input())
num3 = float (input())
if num1==num2==num3:
    print("equilatero")
elif num1==num2 or num1==num3 or (num2==num3):
    print("isosceles")
else:
    print("escaleno")

File: C++/Classificação de Triângulos.cpp
/*
    Autor : Sandeison Silva Fernandes
    Data: 02 / 06 / 2015

*/

#include <iostream>
using namespace std;

int main ( )
{
    float num_1, num_2,num_3;

    cin >> num_1 >> num_2 >> num_3;
    
    if ((num_1 == num_2)&&(num_1==num_3)){
        cout <<"equilatero"<< endl;
    }else{
        if (( num_1 == num_2 )||( num_1==num_3 )||( num_2==num_3 )){
            cout <<"isosceles"<< endl;
        }else{
            cout <<"escaleno"<< endl;
        }


    }

    return 0 ;
}
